RichText-web-module/src/main/webapp/resources/celRteJS/tinyMCE6/loadTinyMCE-async.mjs

```diff
@@ -118,6 +118,9 @@ class CelRteAdaptor {
       console.debug("tinyMceSetupDoneHandler: register 'init' listener for editor", editor.id);
       editor.on('init', (ev) => {
         console.debug("tinyMceSetupDoneHandler: on 'init' for editor done.", editor.id);
+        const origElemClassNames = editor.getElement().className.split(' ')
+          .filter(cssClass => cssClass.startsWith('celEditorBody_'));
+        editor.getBody().className  += ' ' + origElemClassNames.join(' ');
```

Comment:
use `classList` instead of `split`/`join`
```suggestion
        const origElemClassNames = [...editor.getElement().classList]
          .filter(cssClass => cssClass.startsWith('celEditorBody_'));
        editor.getBody().classList.add(...origElemClassNames);
```